src/commands/iota/create-config.ts

```diff
@@ -0,0 +1,170 @@
+    }
+
+    let walletAri: string = flags['wallet-ari'] || ''
+
+    const { wallets } = await cweService.listWallets()
+
+    const walletAris = wallets.map((wallet: any) => wallet.ari)
+    const wrongAriProvided = !walletAris.includes(walletAri)
+
+    if (!walletAri || wallets.length === 0 || wrongAriProvided) {
+      const walletChoices = wallets.map((wallet: any) => ({
+        name: wallet.name || wallet.id,
+        value: wallet.ari,
+      }))
+
+      const CREATE_NEW_WALLET = 'Create new wallet'
+      walletChoices.push({ name: CREATE_NEW_WALLET, value: CREATE_NEW_WALLET })
+
+      const selectedWallet: string = await select({
+        message: 'Select the wallet used to sign the request token to the Affinidi Vault',
+        choices: walletChoices,
+      })
+
+      if (selectedWallet === CREATE_NEW_WALLET) {
+        const walletDidMethodChoices = Object.values(DidMethods).map((method: string) => ({
+          name: method,
+          value: method,
+          default: DidMethods.KEY,
+        }))
+
+        const didMethod = await select({ message: 'Select DID method of wallet', choices: walletDidMethodChoices })
+
+        const newWalletData = {
+          name: validateInputLength(await input({ message: 'Enter wallet name' }), INPUT_LIMIT),
+          description: validateInputLength(
+            await input({ message: 'Enter wallet description (optional)' }),
+            INPUT_LIMIT,
+          ),
+          didMethod,
+          ...(didMethod === DidMethods.WEB && {
+            didWebUrl: validateInputLength(
+              await input({ message: 'Enter did:web URL (your applications domain)' }),
+              INPUT_LIMIT,
+            ),
+          }),
+        }
+
+        const walletSchema = z.object({
+          name: z.string().min(3).max(INPUT_LIMIT).optional(),
+          description: z.string().max(INPUT_LIMIT).optional(),
+          didMethod: z.nativeEnum(DidMethods).optional(),
+          didWebUrl: z.string().max(INPUT_LIMIT).optional(),
+        })
+        const createWalletInput = walletSchema.parse(newWalletData)
+
+        ux.action.start('Creating wallet')
+        const { wallet } = await cweService.createWallet(createWalletInput as any)
+        ux.action.stop('Created successfully!')
+
+        walletAri = wallet.ari
+      } else {
+        walletAri = selectedWallet
+      }
+    }
+
+    const data: CreateIotaConfigurationInput = {
+      name: flags.name ?? validateInputLength(await input({ message: 'Enter Iota configuration name' }), INPUT_LIMIT),
+      description: flags.description ?? '',
+      walletAri,
+      iotaResponseWebhookURL: flags['response-webhook-url'] ?? '',
+      enableVerification:
+        flags['enable-verification'] ?? (await confirm({ message: 'Enable verification?', default: false })),
+      enableConsentAuditLog:
+        flags['enable-consent-audit-log'] ?? (await confirm({ message: 'Enable consent audit log?', default: false })),
```

Comment:
What if the user provided them as false, will this confirm still trigger?

Reply:
I've removed the confirmation input. If flags are not provided the values are `false`, otherwise user can set those like this:

```affinidi iota create-config --name <value> --wallet-ari <value> --enable-consent-audit-log --enable-verification```